packages/venia-ui/lib/components/CartPage/GiftCards/__tests__/__snapshots__/giftCards.spec.js.snap

```diff
@@ -3,18 +3,52 @@
 exports[`it renders correctly when it has cards 1`] = `
 <div>
   <div>
-    <span>
-      unit test code 1
-    </span>
+    <div>
+      <span>
+        unit test code 1
+      </span>
+      <span>
+        Balance: 
+        <span>
+          $
+        </span>
+        <span>
+          99
+        </span>
+        <span>
+          .
+        </span>
+        <span>
+          00
+        </span>
+      </span>
```

Comment:
@sirugh I think this `Price` component doesn't get mocked because we `requireActual` Peregrine in this `.spec.js`. Apparently that will ignore mocks in `__mocks__`?